src/components/searchSnippet/index.jsx

```diff
@@ -75,7 +77,7 @@ function SearchSnippet({ cid, data, mobile, onClickRank }) {
       className="hover-rank"
       display="flex"
       alignItems="center"
-      marginBottom="10px"
+      marginBottom="-2px"
```

Comment:
in general all css props and rules should be covered with classes

Reply:
agree, old code, not possible to refactor everything